File: Praktikum 5. Function/Soal 5/PRAK505-2310817310001-AvantioFierzaPatria.c
#include <stdio.h>

void Biodata(char nama[], char asal[], int tahunLahir)
{
    int tahun_sekarang = 2020;
    int umur = tahun_sekarang - tahunLahir;

    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d tahun\n", umur);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s\n", asal);
}

int main()
{
    int tahunLahir;
    char Namaku[20], Asal[15];

    // Menerima input

    printf("Masukkan nama: ");
    scanf(" %[^\n]%*c", Namaku);

    printf("Masukkan asal: ");
    scanf(" %[^\n]%*c", Asal);

    printf("Masukkan tahun lahir: ");
    scanf("%d", &tahunLahir);

    // Memanggil fungsi Biodata
    Biodata(Namaku, Asal, tahunLahir);

    return 0;
}

File: Praktikum 5. Function/Soal 5/PRAK505-2310817310001-AvantioFierzaPatria.py
def biodata(nama, asal, tahun_lahir):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahun_lahir

    print(f"Perkenalkan Nama Saya : {nama}")
    print(f"Umur Saya : {umur} tahun")
    print(f"Saya Adalah Angkatan : {tahun_sekarang}")
    print(f"Asal Saya dari : {asal}")
    print(f"Tahun Lahir : {tahun_lahir}")


def main():
    nama = input("Masukkan nama: ")
    asal = input("Masukkan asal: ")
    tahun_lahir = int(input("Masukkan tahun lahir: "))

    # Memanggil fungsi biodata
    biodata(nama, asal, tahun_lahir)


if __name__ == "__main__":
    main()
